test/basic-querying.test.js

```diff
@@ -26,6 +26,17 @@ describe('basic-querying', function() {
 
   });
 
+  describe('forceId', function() {
+    it('returns error when default forceId is true', function(done) {
+      User.create({ name: 'a', id: 1 })
+        .then()
```

Comment:
This test will pass when no error is thrown. Remember the golden rule - never trust a unit test you haven't seen fail.

To verify that a method returns rejected promise, one has to do something like this:

```js
User.create({...})
  .then(
    function() { throw new Error('User.create should have failed.'); },
    function(err) { /* assert on err */ });
```

Also if you decide to use promises, then don't mix it with callbacks. You can return a promise from your test function and Mocha will handle it well.

```js
it('...', function() {
  return User.create({...})
    .then(
      function() { throw new Error('User.create should have failed.'); },
      function(err) { /* assert on err */ });
});
```